### PR TITLE
Coffee.size to string

### PR DESCRIPTION
**Pull request by billyfung**
making it easier to display size for wack sites

### CODE REVIEW

**Comment by jinpark** [comment]
The only thing I see as a problem here is that you get the tasting notes in a three line string instead of one line, can you get rid of the new lines and make it 
`papaya, toffee, cherry`
instead of 

```
      papaya, 
      toffee, 
      cherry
```

**Comment by jinpark** [comment]
Also can you put logging statements every time when a coffee is added? So after a run of the scraper we can see, hey 20/20 coffees are added or only 15/20 coffees passed, I should manually update the 5 that failed

File: scrapers/intelli.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (jinpark) [inline scrapers/intelli.py]: why so much whitespace

Comment (billyfung) [inline scrapers/intelli.py]: because im fat and wanted to fit through it

**Comment by jinpark** [comment]
lgtm

**Pull request merged**